daemon/logger/splunk/splunk.go

```diff
@@ -301,7 +490,15 @@ func (l *splunkLogger) postMessage(message *splunkMessage) error {
 }
 
 func (l *splunkLogger) Close() error {
-	l.transport.CloseIdleConnections()
+	l.lock.Lock()
+	defer l.lock.Unlock()
+	if l.closedCond == nil {
+		l.closedCond = sync.NewCond(&l.lock)
+		close(l.stream)
+		for !l.closed {
```

Comment:
Nope, in go `Wait cannot return unless awoken by Broadcast or Signal.`, see https://golang.org/pkg/sync/#Cond.Wait

I use it just as a pattern, just a future proof concept if somebody will use this conditional for something else and not make a mistake.